recipes/q/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "q" %}
+{% set version = "3.1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Sann5/q/archive/pack_issue_326.zip
```

Comment:
Is this the correct URL? We usually get the archive from the tagged version of the official release. This looks more like a fork?